packages/plugins/parser-cache/src/index.ts

```diff
@@ -2,9 +2,16 @@ import { Plugin } from '@envelop/types';
 import { DocumentNode, Source } from 'graphql';
 import lru from 'tiny-lru';
 
+interface Cache<T> {
+  get(key: string): T | undefined;
+  set(key: string, value: T): void;
+}
+
```

Comment:
Maybe we should export this as a DocumentrCache and hard-code the DocumentNode?

Reply:
Ah good idea! 473d892